Descent3/init.cpp

```diff
@@ -1541,6 +1541,13 @@ void InitIOSystems(bool editor) {
   ddio_MakePath(fullname, LocalD3Dir, "extra13.hog", NULL);
   extra13_hid = cf_OpenLibrary(fullname);
 
+  // last library opened is the first to be searched for DLLs, so put this one
+  // at the end to find our newly build script DLLs first
+#ifdef _WIN32
+  ddio_MakePath(fullname, LocalD3Dir, "d3-win.hog", NULL);
+  sys_hid = cf_OpenLibrary(fullname);
+#endif
+
```

Comment:
Better to merge this with other d3-<PLATFORM>.hog loading. After that we can move `sys_hid = cf_OpenLibrary(fullname);` out of #ifdefs.

Reply:
Done.